documentation/modules/rn-2.2.adoc

```diff
@@ -39,3 +51,10 @@ Retaining the `Importer` pod for debug purposes causes warm migration to hang du
 As a temporary workaround, the `Importer` pod is removed at the end of the precopy stage so that the precopy succeeds. However, this means that the `Importer` pod log is not retained after warm migration is complete. You can only view the `Importer` pod log by using the `oc logs -f <cdi-importer_pod>` command during the precopy stage.
 
 This issue only affects the `Importer` pod log and warm migration. Cold migration and the `virt-v2v` logs are not affected.
+
+.Migration plans for VMs with no operation system fail with an unclear error message.
```

Comment:
"operating system"
Recommend you begin a heading with the cause of the problem. It's not the migration plans that are the problem. It's the VMs with no operating system.

Example: Migrating VM with no operating system fails with unclear error message